fallout/gatling/gatling_timeseries_latency.yaml

```diff
@@ -0,0 +1,99 @@
+# master uses a Gatling DSE plugin based on 3.x
+# DSP-17972-unified-driver uses a Gatling DSE plugin based on the unified driver (>= 4.5.x)
+branch: "master"
+
+---
+
+ensemble:
+  observer:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.instance.type: m3.large
+    configuration_manager:
+      - name: ctool_monitoring
+        properties:
+          graphite.create_server: true
+  server:
+    node.count: 3
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.tenant: moonshot-v2
+        cloud.instance.type: ms2.small
+    configuration_manager:
+      - name: ctool
+        properties:
+          product.type: cassandra
+          product.install.type: tarball
+          product.version: "3.11.6"
+      - name: ctool_monitoring
+        properties:
+          components: OS,JVM,GRAPH
+  client:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.instance.type: m3.xlarge
+    configuration_manager:
+      - name: ctool
+        properties:
+          install.maven: true
+      - name: gatling
+        properties:
+          git.repository: git@github.com:riptano/dse-benchmarks.git
+          git.branch: {{branch}}
+
+workload:
+  phases:
+    # This simulation isn't necessary in a single-node case; Timeseries_Write will
+    # create the infrastructure that's needed.  We do need it in the multi-node
+    # client case; otherwise you can see weird timing errors around the existence
+    # of the expected keyspace/table on various nodes.
+    - timeseries-schema:
+        module: gatling
+        properties:
+          maven_args: >
+              scala:testCompile
+              gatling:execute
+              -Dgatling.disableCompiler
+              -Dgatling.simulationClass=core.timeseries.Timeseries_Schema
+    - timeseries-write:
+        module: gatling
+        properties:
+          maven_args: >
+              scala:testCompile
+              gatling:execute
+              -Dgatling.disableCompiler
+              -Dgatling.simulationClass=core.timeseries.Timeseries_Write
+              -DconnectionsPerHost=8
+              -DrowsPerPartition=1000
+              -DinjectionType=constant
```

Comment:
I've been experimenting with throughput parameters [here](https://github.com/riptano/dse-benchmarks/blob/417eb6b411a34d583f970644c1a3825b9f41d14c/src/test/scala/core/ThroughputParameters.scala#L8-L35). It seems `linear-ramp-up` is not a valid value for time series. However `custom-ramp-up` should be close to what we want. I will experiment more when Fallout is back online (it's currently being redeployed).

Reply:
The "custom-ramp-up" injection type is exactly what I was planning on using for the real benchmarks.  In local runs something like the following does do exactly what we'd like:

```
mvn clean scala:testCompile gatling:execute -Dgatling.disableCompiler -Dgatling.simulationClass=core.timeseries.Timeseries_Write -DconnectionsPerHost=8 -DrowsPerPartition=1000 -DinjectionType=custom-ramp-up -Dthroughput.start=10 -Dthroughput.step=10 -Dthroughput.windows.count=3 -Dthroughput.windows.duration=2
```

![07242020_1_start10_step10_count3](https://user-images.githubusercontent.com/3156294/88417568-c5ccfe80-cda7-11ea-9e1a-d12e04c18319.png)